### PR TITLE
fix(server): no min police for safe

### PR DESCRIPTION
**Pull request by mockdot**
## Description

Just a small fix so that it doesn't get overlooked in the future. 

## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My code fits the style guidelines.
- [x] My PR fits the contribution guidelines.

### CODE REVIEW

**Comment by BerkieBb** [comment]
A little indentation can't do any harm 😄

**Comment by mockdot** [comment]
You mean using enters for this line? I thought I was identing with the 4 spaces 😝 , just went ahead and copied the line from the registers one just for a quick fix since I've noticed people abusing this already, with that said I am thinking on porting to lib zones and so on and submitting a proper PR 😁

File: server.lua [hunk 1]
```diff
@@ -130,6 +130,7 @@ RegisterNetEvent('qb-storerobbery:server:trysafe', function()
     local ClosestSafeIndex = GetClosestSafe(PlayerCoords)
 
     if not ClosestSafeIndex then return end
+    if Amount < Config.MinimumCops then if Config.NotEnoughCopsNotify then QBCore.Functions.Notify(PlayerSource, Lang:t('error.no_police', { Required = Config.MinimumCops }), 'error') end return end
```
Comment (Manason) [inline server.lua]:
```suggestion
    if Amount < Config.MinimumCops and Config.NotEnoughCopsNotify then    
        QBCore.Functions.Notify(PlayerSource, Lang:t('error.no_police', { 
            Required = Config.MinimumCops
        }), 'error')
    return
    end
```

**Review by Manason — CHANGES_REQUESTED** [review]

**Comment by mockdot** [comment]
Done

**Review by Manason — APPROVED** [review]

**Comment by ChatDisabled** [comment]
> since I've noticed people abusing this already

Wow already being used in production

**Comment by mockdot** [comment]
> > since I've noticed people abusing this already
> 
> Wow already being used in production

Wait till they find out a wipe is coming once it is all ready

**Pull request merged**